algebird-core/src/main/scala/com/twitter/algebird/CountMinSketch.scala

```diff
@@ -964,7 +972,7 @@ class TopNCMSMonoid[K: Ordering](cms: CMS[K], heavyHittersN: Int = 100) extends
   /**
    * Creates a sketch out of a single item.
    */
-  def create(item: K): TopCMS[K] = TopCMSItem[K](item, cms, params)
+  def create(item: K): TopCMS[K] = TopCMSItem[K](item, cms + item, params)
```

Comment:
I guess this was a bug before. Does the new test find this bug? Why didn't we see a test failure in the past?